### ISSUE TITLE
little flicker after power off

### ISSUE THREAD

**Issue by ecorini** [cnlohr/esp8266ws2812i2s#31]
Hello, Nice work!

After taking off power i noticed that some leds have a flicker (often a little flash to red).
Have you ever had this kind of problem?

Thank you!

**Comment by con-f-use** [comment]
What do you mean by "after taking off power"? If you cut the power connection the LEDs should not light at all. Can you give us steps to reproduce the problem and tell us which commit and branch you're using?

**Comment by ecorini** [comment]
Yes, a moment after taking power off i see a little flash of some leds.

Video:
https://ufile.io/tzix8

Schematic:
[ws2812b_ESP01.pdf](https://github.com/cnlohr/esp8266ws2812i2s/files/1112850/ws2812b_ESP01.pdf)

**Comment by con-f-use** [comment]
I'd guess it's the residual charge on the input capacitors of your LDO and the led-strip. When the esp8266 browns out, it writes a few byte to the daisy-chained LEDs and some of them light up with the residual charge. Not per se a programmatic error in the esp-code rather than a design problem on your side.

Does it occur, when all the LEDs are off and you cut the power? Also you should be fine without the level shifter. 3.3 Volt is enough to trigger the input pin on WS2812s and even if it wasn't an input, the ESP8266 pins are 5V input tolerant (as long as you don't put that across VCC and GND).

**Comment by ecorini** [comment]
It occurs either with LEDs on or with LEDs off before powering off, also tried without level shifter with same behaviour. If i do: power on -> detach ws2812 data wire -> power off  there is no problem.

Thank you for idea,  i will try playing with different value of input capacitors,

(i'm not using all project, but only ws2812_i2s module in anther project)

**Comment by con-f-use** [comment]
Different value caps probably won't do much, as the LED strip has it's own caps in parallel with the LEDs. Rather make sure, the input pin is at a defined state when the power goes out. You might need a pull down of the right value or a more complex switching solution.
